.circleci/datadog/ci-local-config.yaml

```diff
@@ -0,0 +1,119 @@
+# DD_TAGS
+# DD_EXTRA_TAGS
+# DD_ENV
+# apm is explicitly disabled here for cost reasons
+
+
+##################################
+## Log collection Configuration ##
+##################################
+
+## @param logs_enabled - boolean - optional - default: false
+## @env DD_LOGS_ENABLED - boolean - optional - default: false
+## Enable Datadog Agent log collection by setting logs_enabled to true.
+#
+logs_enabled: true
+
+## @param logs_config - custom object - optional
+## Enter specific configurations for your Log collection.
+## Uncomment this parameter and the one below to enable them.
+## See https://docs.*************/agent/logs/
+#
+logs_config:
+
+  ## @param container_collect_all - boolean - optional - default: false
+  ## @env DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL - boolean - optional - default: false
+  ## Enable container log collection for all the containers (see ac_exclude to filter out containers)
+  #
+  container_collect_all: true
+
+  
+####################################
+## Trace Collection Configuration ##
+####################################
+
+## @param apm_config - custom object - optional
+## Enter specific configurations for your trace collection.
+## Uncomment this parameter and the one below to enable them.
+## See https://docs.*************/agent/apm/
+#
+apm_config:
+
+  ## @param enabled - boolean - optional - default: true
+  ## @env DD_APM_ENABLED - boolean - optional - default: true
+  ## Set to true to enable the APM Agent.
+  #
+  enabled: false
+
+######################################
+## Process Collection Configuration ##
+######################################
+
+# @param process_config - custom object - optional
+# Enter specific configurations for your Process data collection.
+# Uncomment this parameter and the one below to enable them.
+# See https://docs.*************/graphing/infrastructure/process/
+
+process_config:
+
+  # @param process_collection - custom object - optional
+  # Specifies settings for collecting processes.
+  process_collection:
+    # @param enabled - boolean - optional - default: false
+    # Enables collection of information about running processes.
+    enabled: false
+
+  # @param container_collection - custom object - optional
+  # Specifies settings for collecting containers.
+  container_collection:
+    # @param enabled - boolean - optional - default: true
+    # Enables collection of information about running containers.
+    enabled: false
+
+  # Deprecated - use `process_collection.enabled` and `container_collection.enabled` instead
+  # @param enabled - string - optional - default: "false"
+  # @env DD_PROCESS_CONFIG_ENABLED - string - optional - default: "false"
+  #  A string indicating the enabled state of the Process Agent:
+  #    * "false"    : The Agent collects only containers information.
+  #    * "true"     : The Agent collects containers and processes information.
+  #    * "disabled" : The Agent process collection is disabled.
+  
+  enabled: "false"
```

Comment:
That deprecated comment lists two other settings to configure. Those other two settings are set to enabled, but this one is set to disabled. Is that correct?

Reply:
My bad, I can't read